srcpkgs/github-cli/template

```diff
@@ -29,9 +29,9 @@ post_install() {
 	CGO_ENABLED=0 GOARCH="" go run script/build.go manpages
 	vcopy share/man usr/share
 
-	GH="${DESTDIR}/usr/bin/gh"
+	local gh="${DESTDIR}/usr/bin/gh"
```

Comment:
from what I can see, this variable is only used once, so I don't think it makes much sense

Reply:
But it doesn't hurt to wrap this variable into local, even just to show that it's a one-off

Reply:
It doesn't hurt, but since we are already improving it, we can get rid of it.

Reply:
Ah, I misunderstood what you requested.